bin/minifollowups/pycbc_page_coincinfo

```diff
@@ -161,25 +165,67 @@ for ifo in files.keys():
 
     time = d['end_time'][:][i]
     utc = lal.GPSToUTC(int(time))[0:6]
+    try:
+        psd_var = d['psd_var_val'][i]
+    except:
+        psd_var = None
 
-    data = [pycbc.results.dq.get_summary_page_link(ifo, utc),
-            str(datetime.datetime(*utc)),
-            '%.3f'  % time,
-            '%5.2f' % d['snr'][i],
-            '%5.2f' % ranking.newsnr(d['snr'][i], rchisq),
-            '%5.2f' % rchisq,
-            '%i'    % d['chisq_dof'][i],
-            '%5.2f' % d['coa_phase'][i],
-            '%5.2f' % bank['mass1'][tid],
-            '%5.2f' % bank['mass2'][tid],
-            '%5.2f' % mchirp,
-            '%5.2f' % bank['spin1z'][tid],
-            '%5.2f' % bank['spin2z'][tid],
-            '%5.2f' % d['template_duration'][i]
-           ]
-    table.append(data)
-
-html += str(pycbc.results.static_table(table, pycbc.results.sngl_table_headers))
+    try:
+        sg_chisq = d['sg_chisq'][i]
+    except:
+        sg_chisq = None
+
+    # This might involve a lot of lines, but it's clear what goes in the output
+    # and in what order.
+    headers = []
+    data.append([])
+
+    # DQ summary link
+    if args.include_summary_page_link:
+        data[-1].append(pycbc.results.dq.get_summary_page_link(ifo, utc))
+        headers.append("Detector&nbsp;status")
+
+    # End times
+    data[-1].append(str(datetime.datetime(*utc)))
+    data[-1].append('%.3f' % time)
+    headers.append("UTC")
+    headers.append("End&nbsp;time")
+
+    # SNR and phase (not showing any single-det stat here)
+    data[-1].append('%5.2f' % d['snr'][i])
+    data[-1].append('%5.2f' % d['coa_phase'][i])
+    #data[-1].append('%5.2f' % ranking.newsnr(d['snr'][i], rchisq))
+    headers.append("&rho;")
+    headers.append("Phase")
+    #headers.append("Stat")
+
+    # Signal-glitch discrimators
+    data[-1].append('%5.2f' % rchisq)
+    data[-1].append('%i' % d['chisq_dof'][i])
+    headers.append("&chi;<sup>2</sup><sub>r</sub>")
+    headers.append("&chi;<sup>2</sup>&nbsp;bins")
+    if sg_chisq is not None:
```

Comment:
any reason not to do the try-except here, which I think would avoid having two switches (try-except earlier and if-then here) and having to define two extra variables?

Reply:
Changes as suggested